SocialCareCaseViewerApi/V1/UseCase/TeamsUseCase.cs

```diff
@@ -21,5 +21,15 @@ public ListTeamsResponse ExecuteGet(ListTeamsRequest request)
 
             return new ListTeamsResponse() { Teams = EntityFactory.ToDomain(teams) };
         }
+
+        public TeamResponse ExecutePost(CreateTeamRequest request)
+        {
+
+            // todo check if team name already exists, if so throw an error
```

Comment:
Would we not want to do this before releasing the ability to create a team? 
Just wondering if there's the risk of duplicate team names being created and then having to sift through them later?

Reply:
There shouldn't be any risk, GetTeams with either team id or team name is following this PR and they should both be in staging together at the same time.

We can wait to merge this until the other PR is ready to go though.

Also the frontend for this isn't done so it's fairly unlikely this backend API is hit by any users for now.